package gameMsg;

message PlayBeginRequest {//玩关卡请求协议
  required string sID = 1;//用户id，微信openid
  required  int32 gateID = 2;//关卡id
}

message PlayBeginResponse {//玩关卡响应协议
  required int32 gateId = 1;//关卡id
  optional int32 nBeginTime = 2;//游戏结束时间
}
